### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by alicehsing**

### CODE REVIEW

File: functions.js [hunk 1]
```diff
@@ -18,7 +18,11 @@ OUTPUT:
 ]*/
 
 export function getDogs(arr) {
-    return [];
+    const getDogsArr = arr.filter((pet) => {
+        if(pet.type === 'dog')
+            return true;
+    });
+    return getDogsArr;
 }
```
Comment (BenFrancka) [inline functions.js:27]: Great solution!  Think about how you could do this with fewer lines of code.  (implicit returns, etc.).

File: functions.js [hunk 2]
```diff
@@ -183,7 +218,10 @@ Output:
 */
 
 export function getChevyCars(arr) {
-    return [];
+    const chevyArr = arr
+        .filter(car => car.type === 'car')
+        .filter(car => car.make === 'chevy');
+    return chevyArr;
```
Comment (BenFrancka) [inline functions.js:224]: Nice job with this!  Remember your comparison operators.  You could make these comparisons in the same filter call if you wanted to: `arr.filter(car => car.type === 'car' && car.make === 'chevy');`

File: functions.js (hunk 1 above)
Comment (alicehsing) [reply functions.js:27]: something like this?!?
       if(pet.type === 'dog' => true);
return getDogsArr;

File: functions.js (hunk 2 above)
Comment (alicehsing) [reply functions.js:224]: Yes, that totally makes it more clean.

**Pull request merged**